qcodes/instrument_drivers/Shamrock/SR750.py

```diff
@@ -0,0 +1,305 @@
+
+        # add the instrument parameters
+        self.add_parameter('blaze',
+                           get_cmd=self.get_blaze,
+                           get_parser=int,
+                           label='blaze')
+
+        self.add_parameter('calibration',
+                           get_cmd=self.get_calibration,
+                           unit='nm',
+                           label='calibration')
+
+        self.add_parameter('grating',
+                           get_cmd=self.get_grating,
+                           set_cmd=self.set_grating,
+                           get_parser=int,
+                           vals=Ints(min_value=1,
+                                     max_value=self.number_gratings),
+                           label='grating')
+
+        self.add_parameter('groove_density',
+                           get_cmd=self.get_groove_density,
+                           get_parser=int,
+                           unit='l/mm',
+                           label='groove density')
+
+        self.add_parameter('slit',
+                           get_cmd=self.get_slit,
+                           set_cmd=self.set_slit,
+                           get_parser=int,
+                           vals=Ints(min_value=10,
+                                     max_value=2500),
+                           unit=u"\u03BC"+'m',
+                           label='slit')
+
+        min_wavelength, max_wavelength = self.ShamrockCIF.get_wavelength_limits(self.device_id, self.get_grating())
+        self.add_parameter('wavelength',
+                           get_cmd=self.get_wavelength,
+                           set_cmd=self.set_wavelength,
+                           get_parser=float,
+                           vals=Numbers(min_value=min_wavelength,
+                                        max_value=max_wavelength),
+                           unit='nm',
+                           label='wavelength')
+
+        # print connect message
+        self.connect_message(idn_param='IDN')
+
+    # get methods
+    def get_blaze(self):
+        grating = self.ShamrockCIF.get_grating(self.device_id)
+        grating_info = self.ShamrockCIF.get_grating_info(self.device_id, grating)
+        return grating_info[1]
+
+    def get_calibration(self):
+        return self.ShamrockCIF.get_calibration(self.device_id, 1024)
+
+    def get_grating(self):
+        return self.ShamrockCIF.get_grating(self.device_id)
+
+    def get_groove_density(self):
+        grating = self.ShamrockCIF.get_grating(self.device_id)
+        grating_info = self.ShamrockCIF.get_grating_info(self.device_id, grating)
+        return grating_info[0]
+
+    def get_idn(self):
+        return {'vendor': 'Shamrock', 'serial': self.serial_number}
+
+    def get_slit(self):
+        return self.ShamrockCIF.get_slit(self.device_id)
+
+    def get_wavelength(self):
+        return self.ShamrockCIF.get_wavelength(self.device_id)
+
+    # set methods
```

Comment:
for these kinds of separators of code, it makes them more readable when they are surrounded by new lines, and not stick to the definition of the next method as it is in this case. So, i'd recommend to add a blank line right here for readablity (and for other "separators" like this).